subsys/bluetooth/audio/bap_scan_delegator.c

```diff
@@ -743,6 +743,14 @@ static int scan_delegator_add_src(struct bt_conn *conn,
 			     subgroup->metadata_len);
 	}
 
+	if (scan_delegator_cbs != NULL && scan_delegator_cbs->add_source != NULL) {
+		err = scan_delegator_cbs->add_source(conn, state);
+		if (err != BT_ATT_ERR_SUCCESS) {
```

Comment:
Ditto here for consistency:
```suggestion
		if (err != 0) {
```